momentum/index.d.ts

```diff
@@ -36,6 +36,77 @@ interface MapData {
 	credits: { type: string, user: { alias: string } }[];
 }
 
+ /***************************************************************************************** */
+
+  interface Region extends JsonObject {
+	points: Vec2D[];
+	bottom: number;
+	height: number;
+	teleportTargetName: string; // mutually exclusive to other two teleport fields
+	teleportPos: Vec3D; // TODO: This below are required if region is part of a volume used by stafe or major checkpoint zone
+	teleportYaw: number; // See convo in mom red 25/09/23 02:00 GMT
+	safeHeight: number;
+  }
+  
+  interface Volume extends JsonObject {
+	regions: Region[];
+  }
+  
+  interface Zone extends JsonObject {
+	volumeIndex: number;
+	filterName: string;
```

Comment:
Should be `filtername` all lowercase, which I went with to match the game code entity key (although entity keys _are_ case-insensitive...)